test/jerboa_test.exs

```diff
@@ -1,35 +1,27 @@
 defmodule JerboaTest do
   use ExUnit.Case, async: true
-  alias Jerboa.Format.Body.Attribute
+
   @moduletag system: true
-  @google_ip {74, 125, 143, 127}
-  @google_port 19_302
 
-  test "binding request and response on UDP" do
-    {:ok, socket} = :gen_udp.open(4096, [:binary, active: false])
+  describe "Jerboa over UDP Transport" do
 
-    :ok = :gen_udp.send(socket, @google_ip, @google_port, binding_request())
-    {:ok, {_, _, response}} = :gen_udp.recv(socket, 0)
-    :ok = :gen_udp.close(socket)
+    test "send binding request; recieve success response" do
 
-    assert {:ok,
-            %Jerboa.Format{
-              class: :success,
-              method: :binding,
-              attributes: [a]}} = Jerboa.Format.decode response
-    assert %Attribute{
-      name: Attribute.XORMappedAddress,
-      value: %Attribute.XORMappedAddress{
-        family: :ipv4,
-        address: {_,_,_,_},
-        port: _}} = a
-  end
+      ## Given:
+      alias Jerboa.Test.Helper.Server
+      alias Jerboa.Test.Helper.Format
+      {:ok, socket} = :gen_udp.open(4096, [:binary, active: false])
+
+      ## When:
+      msg = Jerboa.Format.encode(Format.binding_request())
+      :ok = :gen_udp.send(socket, Server.a(), Server.p(), msg)
+      {:ok, {_, _, response}} = :gen_udp.recv(socket, 0)
+      {:ok, p} = Jerboa.Format.decode(response)
```

Comment:
I'd use `params` instead of `p`. Above we call `Server.p/0`, which may be misleading in this context.